ext/curl/tests/bug77946.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+Bug #77946 (Errored cURL resources returned by curl_multi_info_read() must be compatible with curl_errno() and curl_error())
+--SKIPIF--
+<?php
+
+if (!extension_loaded('curl')) {
+	exit('skip curl extension not loaded');
+}
+
+?>
+--FILE--
+<?php
+$urls = array(
+   'http://nonexistent.tld',
```

Comment:
Is it possible to trigger some other kind of error here, that does not require network? I'm thinking something like `"file://" . __DIR__ . "/does_not_exist"`, if it works.